tools/VersionController/Models/VersionBumper.cs

```diff
@@ -373,25 +441,47 @@ private void UpdateOutputModuleManifest(List<string> releaseNotes)
             var projectModuleManifestPath = _fileHelper.ProjectModuleManifestPath;
             var tempModuleManifestPath = Path.Combine(outputModuleDirectory, moduleName + "-temp.psd1");
             File.Copy(outputModuleManifestPath, tempModuleManifestPath, true);
+
             var script = "$releaseNotes = @();";
             releaseNotes.ForEach(l => script += "$releaseNotes += \"" + l + "\";");
+
+            // Get required module list and update Az,Accounts' version
+            var getRequiredModulesScript = "Import-LocalizedData -BaseDirectory " + outputModuleDirectory + " -FileName " + Path.GetFileName(outputModuleManifestPath) + " -BindingVariable moduleInfo;";
+            getRequiredModulesScript += "$requiredModules = @();";
+            getRequiredModulesScript += "$moduleInfo.RequiredModules.ForEach({ " +
+                        "if ($_.ModuleName -eq \"Az.Accounts\"){ " +
+                        "  $requiredModules += @{ModuleName = \"Az.Accounts\"; ModuleVersion = \"" + _accountsVersion + "\"} " +
+                        "}else " +
+                        "{ " +
+                        "  $requiredModules += $_ " +
+                        "} " +
+                      "});";
```

Comment:
It's recommended to use `@$"..."` to represent a multi-line formatted string. See https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#verbatim-string-interpolation

Reply:
Thank you for sharing this skill.